device_doctor/README.md

```diff
@@ -0,0 +1,72 @@
+# device_doctor
+
+This utility tool is used by LUCI infrastructure to manage device health
+for Flutter LUCI swarming bots.
+
+It offers support for different platforms: linux, mac and windows, and
+different devices: android and iOS.
+
+## Dependencies
+
+Different devices require different tools to be in the `path` beforehand.
+
+### android
+* `adb`
+
+### iOS
+* `idevice_id`
+* `idevicediagnostics`
+* `xcrun`
+
+## Building
+
+This tool is meant to be published as an AOT compiled binary distributed via
+CIPD. To build the AOT binary, run `tool/build.sh` for linux and mac, and run
```

Comment:
Should we instead redirect users to get the latest version from CIPD? There's already a manual build section below

Reply:
Updated.